### PR TITLE
Edits for dev guide on creating kibana dashboards

### PR DESCRIPTION
**Pull request by dedemorton**
@monicasarbu should review this one.

### CODE REVIEW

File: libbeat/docs/newdashboards.asciidoc [hunk 1]
```diff
@@ -60,44 +74,36 @@ Options:
...
+To export all the dashboards for a Beat, including any new or modified dashboards and all dependencies such as
+visualizations, searches, and index patterns, you can use the python script `export_dashboards.py` from  
+https://github.com/elastic/beats/tree/master/dev-tools[dev-tools].
 
-[source,shell]
-----------------------------------------------------------------------
-../dev-tools/import_dashboards.sh -dir etc/kibana
-----------------------------------------------------------------------
+NOTE: Dashboard names must be **prefixed** with the name of the beat in order to work correctly. For example:
+Topbeat-Dashboard.
```
Comment (monicasarbu) [inline libbeat/docs/newdashboards.asciidoc]: Please also add here that before running the `export_dashboards` script the user needs to enable the python environment, that contains all the packages that are required to run the python script.

If you are running the python script for the first time, you need to create the environment by running the following commands, one by one under the `beats/dev-tools` directory:
```
 virtualenv env
 . env/bin/activate
 pip install -r requirements.txt
```
After this, the environment is created under the `env` directory and for the next runs you can just enable the environment by running the command in the `beats/dev-tools` directory:
``` 
 . env/bin/activate
```

**Comment by monicasarbu** [comment]
@dedemorton Thank you Dede. I left a small thing that I think it's good to add to the docs.

**Comment by monicasarbu** [comment]
@dedemorton I opened another PR to add these details to the README file as it's sometimes handy to have this info closer to the source: https://github.com/elastic/beats/pull/1608

**Comment by andrewkroh** [comment]
jenkins, test it

**Pull request merged**